content/en/docs/content-structure/layout-builder/_index.md

```diff
@@ -228,20 +302,58 @@ Gives users the ability to switch page views by selecting tabs across the top of
+| Tab Items             | field_tab_items   | Entity reference revisions | no       | unlimited   |           | Bundle: tab_item    |
 
-### Tab
+### Tab Item
 
-**Machine name:** `tab`
+**Machine name:** `lb_tab_item`
 
 The Tab paragraph is referenced by the Tabs block.
 
 | Label | Machine Name | Type                                 | Required | Cardinality | Help text | Field Settings | Notes |
 |-------|--------------|--------------------------------------|----------|-------------|-----------|----------------|-------|
-| Title | field_title  | Text (plain)                         | yes      | 1           |
-| Body  | field_body   | Text (formatted, long, with summary) | no       | 1           |
+| Heading | field_heading  | Text (plain)                         | yes      | 1           |  |  | H4 |
+| Body  | field_body   | Text (formatted, long, with summary) | no       | 1           |  |  | WYSIWYG |
+
+## Table
+
+- **Machine name:** `lb_table`
+- **Project:** [lb_table](https://www.drupal.org/project/lb_table)
+- **Designs:** [Mobile](<../../../../../assets/img/designs/lb/Tables Mobile.png>) | [Desktop](<../../../../../assets/img/designs/lb/Tables Desktop.png>)
+
+Allows users to be able to view responsive tables within a page.
+
+| Label            | Machine Name | Type                                 | Required | Cardinality | Help text | Field Settings | Notes |
+|------------------|--------------|--------------------------------------|----------|-------------|-----------|----------------|-------|
+| Section heading	   | (inherit)    |  |  |  |  |  | H2 |
+| Section subheading | (inherit)    |  |  |  |  |  | H3 |
+| Body             | field_body         | Text (formatted, long, with summary) | no       | 1           |
+
+## Testimonials
+
+- **Machine name:** `lb_testimonials`
+- **Project:** [lb_testimonial]
+- **Designs:** [Mobile](<>) | [Desktop](<>)
+
+Display of short testimonials or quotes from Y members.
+
+| Label          | Machine Name                     | Type                       | Required | Cardinality | Help text | Field Settings         | Notes |
+|----------------|----------------------------------|----------------------------|----------|-------------|-----------|------------------------|-------|
+| Section heading    | (inherit)     |  |  |  |  |  | H2 |
+| Section subheading | (inherit)     |  |  |  |  |  | H3 |
+| Testimonials | field_testimonial_item | Entity reference revisions | no | 4 |  | Bundle: testimonial_item | 
+
+### Testimonial Item
+
+**Machine Name:** `lb_testimonial_item`
+
+| Label          | Machine Name                     | Type                       | Required | Cardinality | Help text | Field Settings         | Notes |
+|----------------|----------------------------------|----------------------------|----------|-------------|-----------|------------------------|-------|
+| Name | field_item_name | Text (plain) | Yes | 1 |  | Firstname, Lastname | H4 |
```

Comment:
i'm confused by `Firstname, Lastname`.

Reply:
I think I copied it out of the ticket, it's not needed here. Removed.